src/components/dialog/CreateIssueTemplateDialog.vue

```diff
@@ -0,0 +1,852 @@
+<template>
+  <v-dialog v-model="createIssueTemplateDialog" persistent width="auto">
+    <v-card color="surface-elevated-3" rounded="lger" class="pa-3" elevation="0">
+      <v-card-title class="pl-4">Create Issue Template</v-card-title>
+      <v-stepper class="d-flex flex-column" v-model="step" :items="stepLabels" hide-actions bg-color="surface-elevated-3" flat >
+
+        <!-- Step 1: General Info -->
+        <template v-slot:item.1>
+          <v-form v-model="formGeneralValid" validate-on="blur">
+            <v-text-field v-model="templateName" v-bind="templateNameProps" label="Name" class="mb-2" />
+            <v-textarea v-model="templateDescription" v-bind="templateDescriptionProps" label="Description" class="mb-2" />
+            <v-text-field v-model="repositoryURL" v-bind="repositoryURLProps" label="Repository URL" class="mb-2" />
+          </v-form>
+        </template>
+
+        <!-- Step 2: Issue Types, Priorities & States -->
+        <template v-slot:item.2>
+          <v-form v-model="formIssueTypesValid">
+            <v-row>
+                <!-- Issue Types -->
+                <v-col cols="4">
+                    <v-btn
+                        variant="outlined"
+                        block
+                        color="primary"
+                        class="bg-white text-primary rounded-sm px-4 py-2"
+                         @click="() => {createIssueType('', '', '', ''); expandedCardKey = { nameID: '', type: 'type'};}"
+                        >+ Add Issue Type
+                    </v-btn>
+                    <ExpandableCard
+                       v-for="issueType in issueTypes"
+                      :key="issueType.name"
+                      :name="issueType.name"
+                      :description="issueType.description"
+                      :expandedCardKey="expandedCardKey"
+                      type="type"
+                      :nameErrorMessage="nameErrorMessage"
+                      @expand="() => {
+                        expandedCardKey = { nameID: issueType.name, type: 'type' }
```

Comment:
that's a lot of code for inline handlers, maybe extract function in the script section below